AnimatedComponents/src/components/accordion/Accordion.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { Pressable, SafeAreaView, Text, View } from 'react-native'
+import Animated, {
+  useAnimatedStyle,
+  useDerivedValue,
+  useSharedValue,
+  withTiming,
+} from 'react-native-reanimated'
+import { DefaultStyle } from 'react-native-reanimated/lib/typescript/reanimated2/hook/commonTypes'
+import Icon from 'react-native-vector-icons/AntDesign'
+
+import { styles } from './accordion-styles'
+
+interface IAccordionProps {
```

Comment:
use prettier and try not to push code with typescript/ prettier issues.
https://eslint.org/docs/latest/rules/spaced-comment